### PR TITLE
feat: Make Helm Template Dagster work without internal access

### PR DESCRIPTION
**Pull request by tijmenstor**

### CODE REVIEW

File: src/mpyl/steps/deploy/dagster.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
 
 
 class DagsterBase:
+    dagster_helm_chart_version = "1.9.5"
```
Comment (Jorg88) [inline src/mpyl/steps/deploy/dagster.py:36]: Maybe this value should come from the configuration (in the scripting repo)?

Comment (tijmenstor) [reply src/mpyl/steps/deploy/dagster.py]: It's the nicer approach yes, but it's more work (I did it anyway, I just want to whine)

File: src/mpyl/utilities/dagster/__init__.py [hunk 2]
```diff
@@ -28,6 +29,7 @@ def from_dict(config: Dict):
                 global_service_account_override=dagster_config.get(
                     "globalServiceAccountOverride", None
                 ),
+                user_code_helm_chart_version=dagster_config["userCodeHelmChartVersion"],
```
Comment (Jorg88) [inline src/mpyl/utilities/dagster/__init__.py:32]: Maybe add a default version for old tags that don't have this value yet in the config

File: src/mpyl/steps/deploy/dagster.py [hunk 3]
```diff
@@ -204,22 +204,8 @@ def execute(self, step_input: Input) -> Output:
         """
         results = []
         properties = step_input.run_properties
-        context = get_cluster_config_for_project(
```
Comment (Jorg88) [inline src/mpyl/steps/deploy/dagster.py:207]: Feel free to delete these support functions as soon as they're no longer used (hard to check for me)

File: src/mpyl/schema/mpyl_config.schema.yml [hunk 4]
```diff
@@ -375,3 +375,6 @@ definitions:
       globalServiceAccountOverride:
         description: Name of an (external) serviceAccount that will be assigned to a dagster user-deployment in favor of creating a new one
         type: string
+      userCodeHelmChartVersion:
```
Comment (Jorg88) [inline src/mpyl/schema/mpyl_config.schema.yml:378]: You don't want to make this a required value?

File: actions/generate-kubernetes-manifests/action.yaml [hunk 5]
```diff
@@ -29,7 +29,7 @@ inputs:
     required: false
 runs:
   using: docker
-  image: docker://public.ecr.aws/vdb-public/gh-mpyl:v0.17.6
+  image: docker://public.ecr.aws/vdb-public/gh-mpyl:pr-133
```
Comment (Jorg88) [inline actions/generate-kubernetes-manifests/action.yaml:32]: Reminder to revert this before merging!

File: src/mpyl/schema/mpyl_config.schema.yml (hunk 4 above)
Comment (tijmenstor) [reply src/mpyl/schema/mpyl_config.schema.yml:378]: As far as I can tell, none of the other values of Dagster are required, so I am okay with this also not being a required value + we have a default for now.

File: src/mpyl/steps/deploy/dagster.py (hunk 3 above)
Comment (tijmenstor) [reply src/mpyl/steps/deploy/dagster.py:207]: Still in use

File: src/mpyl/schema/mpyl_config.schema.yml (hunk 4 above)
Comment (Jorg88) [reply src/mpyl/schema/mpyl_config.schema.yml:378]: Right sorry, this is shared config between the monorepo and scripting, so it can't be mandatory

**Pull request merged**